src/content/docs/get-started/index.mdx

```diff
@@ -201,7 +165,23 @@ Make sure you are signed into your GitHub account before you start. Otherwise, y
 
 <Task>
 
-### Add Code Sync GitHub app
+### Link repo to content
+
+Edge Delivery Services requires configuration about your site to know what to render. These include at minimum a content reference and a code reference. The simplest approach is to create those references with `fstab.yaml` but you can onboard to Config Service and define the references there if you like.
+
+<Callouts>
+1. Rename [default-fstab.yaml](https://github.com/hlxsites/aem-boilerplate-commerce/blob/main/default-fstab.yaml) to `fstab.yaml`.
+1. Replace `{org}` and `{site}` in `fstab.yaml` using the GitHub url to determine the correct values: `https://github.com/{org}/{site}`.
+1. Commit and push this change to the `main` branch of your repository.
+</Callouts>
+<Aside type="note" title="Config Service Approach">
+  You can onboard instead by using the Config Service via the [Helix Admin API](https://www.aem.live/docs/admin.html). To do this, please follow the steps in the [boilerplate readme](https://github.com/hlxsites/aem-boilerplate-commerce?tab=readme-ov-file#initial-setup).
```

Comment:
Why would they want to use Config service instead of `fstab.yaml`?

Reply:
helix 4's fstab is useful for getting started quickly, since it does not require you to create a helix org.

helix 5's config service is a more scalable, managable approach and provides access to several features that are not possible with helix 4/fstab. Please see the config service documentation for more information: https://www.aem.live/docs/config-service-setup

Reply:
I updated that Aside with an additional link to the https://www.aem.live/docs/config-service-setup.